netbox/resource_netbox_route_targets_test.go

```diff
@@ -0,0 +1,76 @@
+package netbox
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/fbreckle/go-netbox/netbox/client"
+	"github.com/fbreckle/go-netbox/netbox/client/ipam"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	log "github.com/sirupsen/logrus"
+)
+
+func getNetboxRouteTargetsResource(rtName, tenantName string) string {
+	return fmt.Sprintf(`
+resource "netbox_tenant" "rt_acctest_basic" {
+	name = %[2]s
+}
+resource "netbox_ipan_route_targets" "rt_acctest_basic" {
+	name = "%[1]s",
+	description = "rts for acctest",
+	tenant_id = netbox_tenant.test.id
+}`, rtName, tenantName)
+}
+
+func resourceNetboxRouteTargets_test(t *testing.T) {
+	testSlug := "rts"
+	testName := testAccGetTestName(testSlug)
+	resource.ParallelTest(t, resource.TestCase{
+		Providers: testAccProviders,
+		PreCheck:  func() { testAccPreCheck(t) },
+		Steps: []resource.TestStep{
+			{
+
+				Config: getNetboxRouteTargetsResource(testName, testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("netbox_route_targets.rt_acctest_basic", "name", testName),
+					resource.TestCheckResourceAttr("netbox_route_targets.rt_acctest_basic", "description", "rts for acctest"),
```

Comment:
Tests should cover all attributes. Ideally, they should also cover updating the attributes. As mentioned above, I think tenant_id and description cannot be changed right now.

Reply:
thanks for the comments. I have updated the resourceUpdate and corresponding test-cases to cover that case